src/plugins/catalog/catalog.less

```diff
@@ -10,6 +10,18 @@
   list-style: none;
 }
 
+.CatalogDisplay-linkListItem {
+  display: flex;
+  flex-direction: column;
+  margin: 5px 0;
+}
+
+.CatalogDisplay-itemLinkDescription {
+  text-align: left;
```

Comment:
In case of image tiles, `center` would look nicer here (as the title directly above is also centered)

Reply:
Sure thing, wasn't sure which was better. Now it is centered.

Reply:
:+1: